src/gui/util_gui.cpp

```diff
@@ -75,8 +77,8 @@ QValidator::State DoubleValidator::validate(QString& input, int& pos) const
 
 
 
-namespace Util
-{
+namespace Util {
+	
```

Comment:
Stray white space. Even GH displays it. Pro tip: `git show --color` shows stray white space in red.

Reply:
I used `gitk` in this case. I normally use `git`, too, but this change was too long, and GH wasn't available before the push...

Reply:
Np. This remark is merely a result of my text-terminal-centric usage pattern and influence of the Linux kernel coding convention.